.buildkite/pipeline.yml

```diff
@@ -752,17 +752,6 @@ steps:
       queue: central
       slurm_ntasks: 1
 
-  - label: ":computer: 3D sphere limiters deformation flow"
```

Comment:
I would suggest modifying the `label` above as 
```
":computer: 3D sphere deformation flow w/ limiter and FCT"
```
as it is more descriptive since these features were not present in the original deformation flow test case